src/cli/mod.rs

```diff
@@ -170,6 +170,20 @@ enum Command {
         #[clap(short, long)]
         wallet: String,
     },
+
```

Comment:
You need to be more aware of style. No command has an empty line above it. They all have a description.